examples/api/test/widgets/framework/error_widget.0_test.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/framework/error_widget.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Tests ErrorWidget in Debug Mode', (WidgetTester tester) async {
+    final ErrorWidgetBuilder oldBuilder = ErrorWidget.builder;
+    ErrorWidget.builder = (FlutterErrorDetails details) {
+       return ErrorWidget(details.exception);
+    };
+    await tester.pumpWidget(const example.ErrorWidgetExampleApp());
+
+    expect(find.widgetWithText(AppBar, 'ErrorWidget Sample'), findsOne);
+
+    await tester.tap(find.widgetWithText(TextButton, 'Error Prone'));
+    await tester.pump();
+
+    expectLater(tester.takeException(), isInstanceOf<Exception>());
+
+    final Finder errorWidget = find.byType(ErrorWidget);
+    expect(errorWidget, findsOneWidget);
+    final ErrorWidget error = tester.firstWidget(errorWidget);
+
+    expect(error.message, 'Exception: oh no, an error');
+    ErrorWidget.builder = oldBuilder;
+  });
+
+  tearDown(() {
+    ErrorWidget.builder = (FlutterErrorDetails details) {
+      return Container(
+        alignment: Alignment.center,
+        child: Text(
+          'Error!\n${details.exception}',
+          style: const TextStyle(color: Colors.yellow),
+          textAlign: TextAlign.center,
+          textDirection: TextDirection.ltr,
+        ),
+      );
+    };
+  });
+
+  testWidgets(
+    'Tests ErrorWidget in Release Mode', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('Tests ErrorWidget in Release Mode', (WidgetTester tester) async {
```
And update the indentation for this test body accordingly.